client/src/lib.rs

```diff
@@ -139,6 +139,7 @@ impl ApiKeyServiceClient {
     pub async fn make_request(
         &self,
         request: reqwest::Request,
+        http_headers: Vec<(String, String)>,
```

Comment:
The `reqwest::Request` struct already contains headers, so we don't need an extra argument here.